inc/util/string_utils.hpp

```diff
@@ -93,7 +97,17 @@ namespace ebi
         }
         return eol;
     }
+
+    inline bool is_remote_url(const std::string& input)
+    {
+      return boost::starts_with(input, ebi::vcf::HTTP) ||
+             boost::starts_with(input, ebi::vcf::HTTPS) ||
+             boost::starts_with(input, ebi::vcf::FTP) ||
+             boost::starts_with(input, ebi::vcf::FTPS);
```

Comment:
Yes, we do. The callers of `ebi::util::is_remote_url` are checking whether the input string is of URL form. So we have to check the common protocols at the least.

Reply:
But given our use case, do we need to implement such a generic method, or could it be `is_http_url`? (maybe not the best name because it also checks against HTTPS)

Reply:
This method is used in two different ways:
1. to ignore the remote files when looking for the reference accession through `##reference` tag value. In some VCFs, the `##reference` tag value is a remote file location.
2. to check that the URL has one of these four protocols when trying to download something through libcurl utility.
Although we only use `https` for ENA API, I still think we should keep all of these given the usage 1 mentioned above.